src/App.js

```diff
@@ -1,23 +1,76 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import htmlContent from "./content.json";
```

Comment:
It is better to rename htmlContent to HTML_CONTENT.
It means it is const, can't be change.
import HTML_CONTENT from "./content.json";